frontend/src/index.css

```
@@ -54,6 +54,17 @@
   --color-tag-muted: var(--tag-muted);
 }
 
+@theme {
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@theme`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)